examples/plot_knockoff_aggregation.py

```diff
@@ -40,43 +41,40 @@
 plt.rcParams.update({"font.size": 26})
 
 # Number of observations
-n_subjects = 500
+n_subjects = 100
 # Number of variables
-n_clusters = 500
+n_clusters = 100
```

Comment:
This setting defeats the purpose of the experiment: aggregation is almost powerless...

Reply:
You ask me to reduce the number.

Reply:
Sure, but ideally, we would like to fine a regime where results are nice to look at. I agree that this is a bit subtle !
I'll try to play with parameters to see how it goes.